.ci/scripts/setup-arm-baremetal-tools.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright 2024 Arm Limited and/or its affiliates.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# NB: This function could be used to install Arm dependencies
+# Setup arm example environment (including TOSA tools)
+git config --global user.email "github_executorch@arm.com"
```

Comment:
why do we need to do this? I tried to run this script to repro a build issue and unknowingly set the name and email incorrectly on several github commits.

Reply:
Ouch sorry about that.
It just followed along when moved from install_arm() in
.ci/scripts/utils.sh (in the same patch)

I'm not sure if it is needed when running in the CI nowdays, some time back the examples/arm/setup.sh did some git cloning and I assume it might have been needed then.  Ot it could be related to the patch scripts that was used for vela patches.